compiler-plugin/src/main/java/io/ballerina/stdlib/graphql/compiler/ServiceDeclarationAnalysisTask.java

```diff
@@ -64,5 +64,6 @@ public void perform(SyntaxNodeAnalysisContext context) {
         Schema schema = generateSchema(context, interfaceEntityFinder, node, description);
         DocumentId documentId = context.documentId();
         addToModifierContextMap(documentId, node, schema);
+        addToModifierContextMap(documentId, node, serviceValidator.getCacheConfigContext());
```

Comment:
I think we have a local variable declaration analysis task as well. Shouldn't we include this there?

Reply:
It is there, missed it.